src/Parser/Priest/Discipline/CHANGELOG.js

```diff
@@ -1,4 +1,6 @@
 export default `
+07-12-2017 - Added healing per penance bolt in Atonement Source Tab (by Oratio)
```

Comment:
String-based changelog style is deprecated. Please convert to the new format (see other specs/core) when possible.

Reply:
Oh nice new feature. I'll update the complete log in another PR right after this